ubuntu/ubuntu-14.04-amd64.json

```diff
@@ -284,17 +294,16 @@
     "https_proxy": "{{env `https_proxy`}}",
     "hyperv_generation": "2",
     "hyperv_switch": "{{env `hyperv_switch`}}",
-    "iso_checksum": "946a6077af6f5f95a51f82fdc44051c7aa19f9cfc5f737954845a6050543d7c2",
+    "iso_checksum": "07e4bb5569814eab41fafac882ba127893e3ff0bdb7ec931c9b2d040e3e94e7a",
     "iso_checksum_type": "sha256",
-    "iso_name": "ubuntu-14.04.1-server-amd64.iso",
+    "iso_name": "ubuntu-14.04.4-server-amd64.iso",
```

Comment:
This is intentionally not bumped because it introduces the HWE kernel which generally isn't recommended for virtualized hosts. We've debated just going with HWE but it'd be a change in behaviour so we'd want to make sure this doesn't cause an issue.

Reply:
so if you install from a later version, you get the HWE kernel by _default_ and the only way to prevent it is using old install media - upgrading doesn't apply HWE if installed in this manner

Reply:
Relevant upstream link: https://wiki.ubuntu.com/Kernel/LTSEnablementStack#Ubuntu_14.04_LTS_-_Trusty_Tahr